src/sentry/api/endpoints/organization_member_team_details.py

```diff
@@ -292,4 +292,4 @@ def delete(
             )
             omt.delete()
 
-        return Response(status=200)
+        return Response(serialize(team, request.user, TeamWithProjectsSerializer()), status=200)
```

Comment:
Do we know if anything relies on the current response format. Will the new body break them?

Reply:
Right now there is no response